src/main/java/HtmlUtil.java

```diff
@@ -1,50 +1,40 @@
 import fitnesse.responders.run.SuiteResponder;
 import fitnesse.wiki.*;
 
-
 public class HtmlUtil {
+	private static String SETUP_NAME = "SetUp";
```

Comment:
Useful because the variables are used multiple times; less error-prone in case of naming changes